src/app/components/ContactsList.js

```diff
@@ -53,12 +51,13 @@ const ContactsList = ({ contacts, onCheck, history, contactID, location }) => {
         key
     }) => {
         if (canMerge && !index) {
-            return <MergeRow key={key} style={style} />;
+            return (
+                <MergeRow key={key} style={style} mergeableContacts={mergeableContacts} userKeysList={userKeysList} />
+            );
         }
-
         const contactIndex = canMerge ? index - 1 : index;
 
-        const { ID, Name, LabelIDs = [], emails, isChecked } = contacts[contactIndex];
```

Comment:
I chose to define `emails` to make a difference between what is returned by the API and what we set from the FE side. Keep this logic please.

Reply:
There was some conflict with that at some point in a different file, and then I thought it was a mistake everywhere. It's corrected now!